cfgov/v1/jinja2/v1/story_page.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout-full.html" %}
+
+{% block hero %}
+    {% for block in page.header %}
+        {% if block.block_type == 'features' %}
+            <section class="wrapper
+                            wrapper__match-content
+                            block
+                            block__bg
+                            block__border
+                            block__flush-top
+                            u-mb0">
+                {% include_block block %}
+            </section>
+        {% else %}
+            {% include_block block %}
+        {% endif %}
+    {% endfor %}
+{% endblock %}
+
+{% block content_modifiers -%}
+  {{ super() }} content__center-image-bleed
+{%- endblock %}
+
+{% block content_main %}
+    {% for block in page.content -%}
+        {% if 'featured_content' in block.block_type %}
+            <div class="block{{ ' block__flush-top' if loop.first else '' }}">
+                {% include_block block %}
+            </div>
+        {% else %}
+            {% import 'templates/render_block.html' as render_block with context %}
+            {{ render_block.render(block, loop.index) }}
+        {% endif %}
+    {%- endfor %}
+{% endblock %}
```

Comment:
Did some testing and I am pretty confident this can be simplified to just this:

```suggestion
{% block content_main %}
    {% for block in page.content %}
        <div class="block{{ ' block__flush-top' if loop.first else '' }}">
            {% include_block block %}
        </div>
    {% endfor %}
{% endblock %}
```

(`render_block` is an older custom method that should eventually be completely replaced by the Wagtail native `include_block`)

Reply:
TY TIL. I'll pull it down and verify nothing's borked.